### ISSUE TITLE
Topic 15: Loss of Focus during keyboard navigation

### ISSUE THREAD

**Issue by HD508Github** [Section508Coordinators/DHSA-TT-200-A#12]
Within Topic 15 > Lesson 1: Meaningful Content (::before ::after) > Failing Examples page > Example 1 - Interactive Sample, visual focus is lost during keyboard navigation.  Specifically, loss of focus occurs between the iFrame and the 'ANDI' interactive logo.  This issue also exists for Example 2 - Interactive Sample, Example 3 - Interactive Sample and on the Lesson 2: Content Order and Meaning (CSS position) > Passing examples page > Example 1 - Interactive Sample.
![Top15_Loss of focus (Lesson1-Failing Examples page)](https://user-images.githubusercontent.com/34749031/58199256-16c1fe00-7c9e-11e9-8b21-dfdece1f01e0.jpg)

**Comment by drewnielson** [comment]
The loss of focus is on the iframe itself. If you can confirm that you tested in IE, then we can confirm that this is not a bug. IE does not display visual focus on iframes. Other browsers do, so not an issue with the content but with the browser. We will need to add a note to the Keyboard topic to point this out though.

**Comment by HD508Github** [comment]
The iFrames were tested using Firefox and not IE.  However, I ran the test in IE and the issue doesn’t exist (visual focus appears on the iFrames and there’s no loss of visual focus).  Additionally, I ran the test using Chrome and Chrome doesn’t display focus on iFrames at all.

Kind Regards,
Halimatou(Halima)Diallo
Sr. Accessibility Technician
Supporting the Department of Homeland Security|MGMT/OCIO/OAST
Office: 202-815-5957
Email: Halimatou.Diallo@associates.hq.dhs.gov<mailto:Halimatou.Diallo@associates.hq.dhs.gov>
DHS Accessibility Help Desk: 202-447-0440 (Voice), 202-447-0582 (Fax)
Website: accessibility.dhs.gov<http://accessibility.dhs.gov/> | Email: accessibility@hq.dhs.gov<mailto:accessibility@hq.dhs.gov>
[cid:image001.png@01D3A09F.E6927B00]
This communication, along with any attachments, is covered by federal and state law governing electronic communications and may contain sensitive and legally privileged information. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, use or copying of this message is strictly prohibited.  If you have received this message in error, please reply immediately to the sender and delete this message.  Thank you.

**Comment by drewnielson** [comment]
Right - so no visual focus on the iframes in Chrome and Firefox, but IE and Edge display a visual outline when the iframe has focus. Different behavior in different browsers, and unfortunately not something that we can effectively influence in code to ensure the same behavior across browsers.
